### PR TITLE
include SVG solution information

### PR DESCRIPTION
**Pull request by rboucher**
update to include information about saving SVG from solutions. Comment made on 3/16 by a customer asking about this. 

copied below..
Goodforallpaul 
Are you going to update the icons so that they mirror the ones used on the azure.com website https://azure.microsoft.com/en-gb/solutions/architecture/

### CODE REVIEW

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/diagrams.md](https://github.com/rboucher/architecture-center/blob/patch-1/docs/resources/diagrams.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/resources/diagrams?branch=pr-en-us-460) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C20%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201803200812326818-460%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vVrxb%2Bb7reuTSHZk9U%2BScx1LNQviC4vVmW%2F8Ir4Trsg%3D&st=2018-03-20T08%3A09%3A15Z&se=2018-04-20T08%3A14%3A15Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/resources/diagrams.md [hunk 1]
```diff
@@ -25,6 +25,10 @@ Additional symbols for Microsoft Office and related technologies are available i
 
 A version of the diagrams used in the [Reference Architectures](../reference-architectures/index.md) is available for [download in Visio format](https://aka.ms/arch-diagrams).
 
+## Solution Architecture icons
+
+Microsoft publishes [solution architectures and accompanying diagrams](https://azure.microsoft.com/en-us/solutions/architecture/). The diagrams are in downloadable in SVG format. SVG can be opened and then modified by many tools, including Visio and PowerPoint. If you upgroup the diagram, you can get at the individual icons.  To download a diagram, right-click on it and choose "Save As". You should see an option to save as SVG. If you do not see this option, try again with a different browser.
```
Comment (bennage) [inline docs/resources/diagrams.md]: We need to remove the `/en-us/` from the URL (and  test it because some times it won't work 🤕 ).

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/diagrams.md](https://github.com/rboucher/architecture-center/blob/patch-1/docs/resources/diagrams.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/resources/diagrams?branch=pr-en-us-460) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C21%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201803211922242987-460%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=%2Fq1xxdcO25WCyvw7%2BAXPmfP7Mws4pistDgtTr%2BJZkDs%3D&st=2018-03-21T19%3A19%3A32Z&se=2018-04-21T19%3A24%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/resources/diagrams.md (hunk 1 above)
Comment (rboucher) [reply docs/resources/diagrams.md]: Changed made.

**Comment by rboucher** [comment]
@bennage . I've made the changes you requested.  en-us is removed. Is there something else to do?

**Comment by MikeWasson** [comment]
@rboucher We already link to the Azure Icon Set, which includes SVGs as well as Visio stencils for most Azure services. It looks like the Solution Architecture diagrams are using those icons. 

But it's not a bad idea to link to those diagrams. However I'm thinking the heading should be "Solution Architecture diagrams", mention that users can download the diagrams as SVGs, w/out going into the icons (which are already available via the Icon Set).

File: docs/resources/diagrams.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ Additional symbols for Microsoft Office and related technologies are available i
 
 A version of the diagrams used in the [Reference Architectures](../reference-architectures/index.md) is available for [download in Visio format](https://aka.ms/arch-diagrams).
 
+## Solution Architecture icons
+
+Microsoft publishes [solution architectures and accompanying diagrams](https://azure.microsoft.com/solutions/architecture/). The diagrams are in downloadable in SVG format. SVG can be opened and then modified by many tools, including Visio and PowerPoint. If you upgroup the diagram, you can get at the individual icons.  To download a diagram, right-click on it and choose "Save As". You should see an option to save as SVG. If you do not see this option, try again with a different browser.
```
Comment (MikeWasson) [inline docs/resources/diagrams.md]: upgroup s/b ungroup?

**Comment by rboucher** [comment]
@MikeWasson Actually Mike, the Solution Architecture diagrams don't necessarily use the Azure icon set.  There is some overlap, but the Solution diagrams they tend to make up some combinations of icons as well.  As the originator of the Azure icon set, I know this for sure. Though, I'm happy to change the title of the section.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/diagrams.md](https://github.com/rboucher/architecture-center/blob/patch-1/docs/resources/diagrams.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/resources/diagrams?branch=pr-en-us-460) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C28%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201803280809182742-460%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gs%2BApbNPTIYQsn4ZJ9V0dJWzl%2FbnbqEGGasW%2Fgpy4KY%3D&st=2018-03-28T08%3A05%3A20Z&se=2018-04-28T08%3A10%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bennage** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/resources/diagrams.md](https://github.com/rboucher/architecture-center/blob/patch-1/docs/resources/diagrams.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/azure/architecture/resources/diagrams?branch=pr-en-us-460) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C28%5C0659c0de-90b3-6253-8249-f3f6e8fba551%5CPullRequest%5C201803281941437618-460%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=sRvcAmV42ZDdxfS9xdGKILm4FKrTBrzb4D3eXC3hqyY%3D&st=2018-03-28T19%3A37%3A43Z&se=2018-04-28T19%3A42%3A43Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by MikeWasson** [comment]
@rboucher Makes sense, thanks. Last thing is that I removed the language about right-clicking the diagrams because that may be browser / OS specific.

**Pull request merged**